### PR TITLE
Fix the CI/CD pipeline

### PR DESCRIPTION
**Pull request by ubajze**

### CODE REVIEW

**Review by dgarros — APPROVED** [review]
LGTM, @jvanderaa I would love your input too if you have some time to review

**Comment by dgarros** [comment]
and thank you @ubajze for taking care of this issue so quickly, much appreciated

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,41 +11,21 @@ jobs:
...
-      - name: "Install Python Packages for Linting"
-        run: "pip install --upgrade pip wheel invoke toml"
-      - name: "Install Poetry Packages"
-        run: |
-          pip install poetry
-          poetry config virtualenvs.create false
-          poetry config installer.parallel false
-          poetry install --no-interaction --no-ansi --no-root
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v2"
```
Comment (jvanderaa) [inline .github/workflows/ci.yml:15]: Do we need this to include Python pinned to version `3.9.7`?

**Review by jvanderaa — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jvanderaa) [reply .github/workflows/ci.yml:15]:
```yaml
- name: "Setup environment"
  uses: "networktocode/gh-action-setup-poetry-environment@v2"
    with:
      python-version: "3.9.7" 
```

**Pull request merged**